plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -27,41 +27,50 @@
 {%- endmacro -%}
 
 {% macro bigquery_table_options(persist_docs, temporary, kms_key_name) %}
-  {% set opts = {} %}
+  {% set opts = {} -%}
```

Comment:
I think you're going to need to add `labels` to the macro signature here! Is `labels` on line 42 below currently defined?

Reply:
Good catch!